0015-ilp-addresses/0015-ilp-addresses.md

```diff
@@ -0,0 +1,123 @@
+# ILP Addresses
+
+_ILP addresses_ provide a way route payments to their intended destination through a recursive series of hops, including any number of ILP Connectors. (This happens after the payment is set up on a higher level, such as [SPSP](../0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md).) Addresses can be subdivided into two categories:
```

Comment:
It might be worth adding some verbiage somewhere in here that ILP addresses are not meant to be user-facing (assuming that's still the case).  For example, here was some feedback from the mailing list from a few weeks back:

> Our experience so far is that the most confusing thing about ILP Addresses is that they are ASCII characters, whereas IP Addresses are just numbers. Many people have interpreted that to mean that ILP Addresses are user-facing. They are not! They are similarly ephemeral and low-level as IP Addresses, and a user should never see one. They are used for routing and nothing else.